pkg/grapicmd/context.go

```diff
@@ -0,0 +1,76 @@
+package grapicmd
+
+import (
+	"io"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+	"github.com/spf13/viper"
+
+	"github.com/izumin5210/grapi/pkg/grapicmd/protoc"
+	"github.com/izumin5210/grapi/pkg/grapicmd/util/fs"
+)
+
+type Ctx struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Ctx should have comment or be unexported

Reply:
fixed 👌